src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/resources/accounts.js

```diff
@@ -12,6 +12,7 @@ angular.module('platformWebApp').factory('platformWebApp.accounts', ['$resource'
         lock: { url: 'api/platform/security/users/:id/lock', method: 'POST' },
         getUserApiKeys: { url: 'api/platform/security/users/:id/apikeys', method: 'GET', isArray: true},
         saveUserApiKey: { url: 'api/platform/security/users/apikeys', method: 'POST' },
-        deleteUserApiKey: { url: 'api/platform/security/users/apikeys', method: 'DELETE' }
+        deleteUserApiKey: { url: 'api/platform/security/users/apikeys', method: 'DELETE' },
+        verifyEmail: { url: 'api/platform/security/users/:id/verifyEmail', method: 'POST' }
```

Comment:
Why id here? API waiting for userName.

Reply:
![image](https://user-images.githubusercontent.com/47894313/109147304-861b2a80-776d-11eb-8395-9b383e14924a.png)

Reply:
renamed to userId in all places